lib/src/components/NxSegmentedButton/NxSegmentedButton.tsx

```diff
@@ -46,13 +47,15 @@ const NxSegmentedButton = forwardRef<HTMLDivElement, Props>(
 
       return (
         <div ref={ref} className={classes} onKeyDown={onKeyDown} { ...attrs }>
-          <NxButton variant={variant}
+          <NxButton type={type || undefined}
```

Comment:
Rather than `undefined` would we be safer to make the default `button`?

Reply:
> Rather than undefined would we be safer to make the default button?

I decided against that because we try to make our button components behave similar to HTML `button`.  `NxButton` deliberately stays consistent with `<button>` in regards to the `type` attr, and I think `NxSegmentedButton` should too.